README.md

```diff
@@ -166,14 +166,12 @@ will arbitrarily pick the first one. This can lead to an inaccurate `branch` val
 
 ### Buddy
 
-For builds triggered when [a Pull Request is opened/updated](https://buddy.works/knowledge/deployments/pull-requests),
-Buddy doesn't provide an environment variable indicating the branch from which the Pull Request originated nor the
-target branch. It also build from a branch named `pull/<PR number>` so the target branch cannot be determined with
-a `git` command.
-Therefore, in the case of Pull Request builds, `env-ci` will not be able to determine the `branch` and `prBranch`
-properties.
+For builds triggered when [a Pull Request is opened/updated](https://buddy.works/docs/pipeline-examples/pull-request-testing),
```

Comment:
unless i'm misunderstanding, these environment variables should simply be used by the buddy service implementation within env-ci, correct? if so, please also suggest the appropriate change to https://github.com/semantic-release/env-ci/blob/master/services/buddy.js.

that change would make buddy no longer a caveat. because of that, i think it would be better to just remove this section

Reply:
You are right.